packages/filebrowser-extension/src/index.ts

```diff
@@ -297,9 +297,19 @@ function activateBrowser(
       if (navigateToCurrentDirectory) {
         const { newValue } = change;
```

Comment:
Ooh, found another thing: if `newValue` is null (such as is the case if the last document is closed), then `contextForWidget` errors. We should check for `null` and return early in that case.